crm_operating_unit/views/crm_lead_view.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0"?>
+<!-- Copyright 2015 Eficent Business and IT Consulting Services S.L.
+     Serpent Consulting Services Pvt. Ltd.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<openerp>
+    <data>
+
+        <!-- CRM Lead Form View  -->
+        <record model="ir.ui.view" id="crm_case_form_view_leads">
+        <field name="name">CRM - Leads Form</field>
+        <field name="model">crm.lead</field>
+        <field name="inherit_id" ref="crm.crm_case_form_view_leads"/>
+        <field name="arch" type="xml">
+            <field name="probability" position="before">
+                <field name="operating_unit_id" groups="operating_unit.group_multi_operating_unit" options="{'no_create': True}"/>
+            </field>
```

Comment:
In the leads form. I would place the operating_unit_id field in the tracking group, under the company, instead of in the Opportunities group. Apart from this, the rest looks fine to me.